test/s3-e2e/s3.go

```diff
@@ -0,0 +1,111 @@
+// +build test
+
+package s3_e2e
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awscfg "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"strings"
+)
+
+func CreateBucket(bucket string) func() {
```

Comment:
can you make sure you delete testapi/minio.go if we don't need it's endpoints anymore

Reply:
or alternatively you might want to use that code directly

Reply:
done